packages/commonwealth/server/routes/generateImage.ts

```diff
@@ -57,8 +57,10 @@ const generateImage = async (
   let image;
   try {
     const response = await openai.images.generate({
+      model: 'dall-e-3',
```

Comment:
1 more instance we can update to use this model

https://github.com/hicommonwealth/commonwealth/blob/373f274d5174ea1d359e05a8ac5e03b40b8fe87b/libs/model/src/services/openai/generateTokenIdea.ts#L136-L141